### ISSUE TITLE
league/fractal ^0.19 dependency conflicts with Element API plugin

### ISSUE THREAD

**Issue by jamesmacwhite** [riasvdv/craft-scout#146]
@riasvdv Just making you aware, the recent 2.2.0 release pushed an update to the league/fractal dependency, however there is a bit of  a conflict with the Element API plugin as that is currently specifying ^0.18.

```
  Problem 1
    - Installation request for rias/craft-scout ^2.2 -> satisfiable by rias/craft-scout[2.2.0].
    - Conclusion: remove league/fractal 0.18.0
    - Conclusion: don't install league/fractal 0.18.0
    - rias/craft-scout 2.2.0 requires league/fractal ^0.19 -> satisfiable by league/fractal[0.19, 0.19.1, 0.19.2].
    - Can only install one of: league/fractal[0.19, 0.18.0].
    - Can only install one of: league/fractal[0.19.1, 0.18.0].
    - Can only install one of: league/fractal[0.19.2, 0.18.0].
    - Installation request for league/fractal (locked at 0.18.0) -> satisfiable by league/fractal[0.18.0].
```

**Comment by jamesmacwhite** [comment]
I guess one workaround is to stay on 2.1.4 temporarily

**Comment by riasvdv** [comment]
Should be fixed in 2.2.1, thanks for bringing it to my attention!

**Comment by jamesmacwhite** [comment]
Thanks for the quick fix!
